specification/network/resource-manager/Microsoft.Network/stable/2024-03-01/virtualNetwork.json

```diff
@@ -2021,6 +2021,11 @@
             "$ref": "./networkWatcher.json#/definitions/FlowLog"
           },
           "description": "A collection of references to flow log resources."
+        },
+        "privateEndpointVNetPolicies": {
+          "readOnly": true,
```

Comment:
Is this a read only property? Or should customer be able to change the value?

Reply:
Fixed